test/tests/float.py

```diff
@@ -88,3 +88,16 @@ class F2(float):
     float(-l)
 except OverflowError as e:
     print e.message
+
+all_args = [None, 1, 1L, -1, -1L, 2.0, 0.5, 0]
+for lhs in all_args:
+    for rhs in all_args:
+        for mod in all_args:
+            try:
+                pow(lhs, rhs, mod)
+            except ValueError as e:
```

Comment:
Can we print out the type of the exceptions here?  Maybe something like
```
except Exception as e:
  print type(e), e
```
It looks like we are generating some ValueErrors where cpython generates TypeErrors.

Also, can we print out the result of the `pow` call?  Also let's add some more entries to all_args while we're at it :)  Maybe add `["", 0.0, -0.0, -0.5, float('nan'), float('inf')]` (looks like `-0.5` generates the wrong error message and `-0.0` produces a slightly bad result [-0.0 instead of 0.0]).